app/.storybook/config.ts

```diff
@@ -1,11 +1,17 @@
-import { configure } from '@storybook/react';
+import { configure, addDecorator, addParameters } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import { withCssResources } from '@storybook/addon-cssresources';
 import requireContext from 'require-context.macro';
-import '../node_modules/bootstrap/dist/css/bootstrap.css'
 
-const req = requireContext('../tests', true, /\.stories\.tsx?$/);
+addDecorator(withInfo);
+addDecorator(withCssResources);
+addParameters({
+  cssresources: [{
+    id: 'bootstrap',
+    code: `<link rel="stylesheet" type="text/css" href="/static/bootstrap.min.css"></link>`,
+    picked: true,
+  }]
+})
```

Comment:
This needs some linting. Somehow XO isn't running in the `.storybook` directory

Reply:
Interesting...I'll look at that

Reply:
Even explicitly calling `xo` with full paths to storybook typescript files, eg. `yarn xo .storybook/*` there are no linter errors. What were you expecting?

Reply:
Shouldn't have spaces in the curly braces when importing a named import. Missing semicolon in the call to `addParameters`. Extraneous comma in `picked: true,`